addons/l10n_nl/data/account_fiscal_position_tax_template.xml

```diff
@@ -124,6 +144,11 @@
             <field name="tax_src_id" ref="btw_6_buy"/>
             <field name="tax_dest_id" ref="btw_I_6"/>
         </record>
+        <record id="position_tax_intracom_9" model="account.fiscal.position.tax.template">
```

Comment:
For the intracom (EU Countries) you are missing the mapping for:

`position_tax_intracom_2`

and

`position_tax_intracom_d_2`